libr/core/anal_objc.c

```diff
@@ -182,23 +188,36 @@ static bool objc_find_refs(RCore *core) {
 
 	int total = 0;
 	ut64 off;
+	bool readSuccess = true;
 	for (off = 0; off < objc._data->vsize ; off += objc2ClassSize) {
```

Comment:
```suggestion
	for (off = 0; off < objc._data->vsize && readSuccess; off += objc2ClassSize) {
```
I think this it makes it easier to understand `readSuccess` makes the loop interrupt, instead of having it in a separate `if` just below.